docs/providers/aws/guide/building.md

```diff
@@ -12,6 +12,8 @@ In Serverless Framework V.4, [esbuild](https://github.com/evanw/esbuild) is incl
 
 By default, if your AWS Lambda handler is using Typescript files directly, the Framework will build your code automagically upon deploy, without a plugin. No configuration is necessary by default.
 
+If your AWS Lambda handler is using ESM syntax in a `.js` file, please make sure you remove the "type" property from your "package.json" file and configure the build settings in your `serverless.yml` file as shown below.
+
```

Comment:
@Mmarzex I was under the impression that we build automatically when the user is using ESM syntax in their `.js` files, but that doesn't seem to be the case. It would be better if we could do that just like TypeScript, otherwise the `build` property is no longer optional.

Reply:
Why would we build/bundle ESM by default? ESM is a supported syntax in Lambda for the last few years. If this is a requirement for dev mode right now then there is something wrong with dev mode and how it handles ESM based handlers.

Reply:
That used to work prior to the build work as per the test files, so not sure what happened there.